index.js

```diff
@@ -55,7 +55,15 @@ const register = (server, option) => {
             if (auth.isAuthenticated) {
                 // Credentials also have: .expiresIn, .token, .refreshToken
                 // Put the Auth0 profile in a cookie. The browser may ignore it If it is too big.
-                request.cookieAuth.set({ user : auth.credentials.profile });
+                if (auth.credentials.profile.raw.scope) {
```

Comment:
There is no need for the `if ()` statement here. The `scope` either exists or it doesn't, the `if` doesn't do anything.